src/code/sys_cfb.c

```diff
@@ -1,11 +1,11 @@
 #include "global.h"
 
-u32 sSysCfbFbPtr[2];
-u32 sSysCfbEnd;
+uintptr_t sSysCfbFbPtr[2];
+uintptr_t sSysCfbEnd;
 
 void SysCfb_Init(s32 n64dd) {
     u32 screenSize;
```

Comment:
`screenSize` may be `size_t`? it's the length of the 1D'ed color frame buffer array